docs/operate/mobility/define-geometry.md

```diff
@@ -7,3 +7,179 @@ type: "docs"
 no_list: true
 description: "Specify your robot's dimensions and how it is positioned in space."
 ---
+
+Before you can use the motion planning or navigation services, you need to create a description of your machine's dimensions and how it is positioned relative to its surroundings.
```

Comment:
to center the functionality rather than the service
```suggestion
Before you can use the motion planning or navigation with your machine, you need to create a description of your machine's dimensions and how it is positioned relative to its surroundings.
```